controls/radgridview/styles-and-templates/columns-styling-checkboxcolumn.md

```diff
@@ -1,194 +1,357 @@
+                                    BorderThickness="1"
+                                    BorderBrush="{TemplateBinding Background}">
+                                <Border x:Name="InnerBorder" BorderThickness="1" BorderBrush="{StaticResource CheckBoxInnerBorder_Normal}" Background="{StaticResource CheckBoxInnerBackground_Normal}"/>
+                            </Border>
+                            <Path x:Name="CheckIcon"
+                                    FlowDirection="LeftToRight"
+                                    Visibility="Collapsed"
+                                    Width="9"
+                                    Height="9"
+                                    Stretch="None"
+                                    StrokeThickness="1.5"
+                                    Stroke="{StaticResource CheckBoxTickStroke_Normal}"
+                                    Data="M1.5,6 L3.5,7.5 L7,2"/>
+                            <Path x:Name="IndeterminateIcon"
+                                    FlowDirection="LeftToRight"
+                                    Visibility="Collapsed"
+                                    Width="9"
+                                    Height="9"
+                                    Stretch="None"
+                                    StrokeThickness="1.5"
+                                    Stroke="{StaticResource CheckBoxTickStroke_Normal}"
+                                    Data="M2,4 L 7,4"/>
+                        </Grid>
+                        <ContentPresenter x:Name="ContentItem"
+                                Grid.Column="1"
+                                Margin="{TemplateBinding Padding}"
+                                Content="{TemplateBinding Content}"
+                                ContentTemplate="{TemplateBinding ContentTemplate}"
+                                HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                                RecognizesAccessKey="True"/>
+                    </Grid>
+                </ControlTemplate>
+
+
+
+The customized __ControlTemplate__ can be applied as follows.
+
+#### __[XAML] Example 4: Applying the ControlTemplate to CheckBox__
+
+	<Style x:Key="CheckBoxStyleGridViewColumn" TargetType="CheckBox">
+    	<Setter Property="Template" Value="{StaticResource CheckBoxTemplate}"/>
+    </Style>
+
+    <Style TargetType="CheckBox" BasedOn="{StaticResource CheckBoxStyleGridViewColumn}"/>
+
+## Applying the Styles to GridViewCheckBoxColumn explicitly
+
+The aforementioned approach implies defining the styles __implicitly__. If applying the modifications to a particular instance of __RadGridView__ is needed, these Styles can be applied __explicitly__ through the __Resources__ of __RadGridView__.
+
+	<telerik:RadGridView.Resources>
+		...
+	</telerik:RadGridView.Resources>
+
+## Centering GridViewCheckBoxColumn in Edit Mode

```

Comment:
It looks like this section covers both edit and view mode. Is the title correct?